lua-tree/share/lua/5.1/kong/tools/batch_queue.lua

```diff
@@ -0,0 +1,327 @@
+-- Batch queue with background retryable processing
```

Comment:
is this file part of a different patch?  i don't see where it's called from.
(not an objection, just trying to understand where does it fit)